Makefile

```diff
@@ -64,12 +64,13 @@ upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -qr requirements/pip-tools.txt
 	# Make sure to compile files after any other files they include!
+	$(PIP_COMPILE) --allow-unsafe --rebuild -o requirements/pip.txt requirements/pip.in
 	$(PIP_COMPILE) --rebuild -o requirements/pip-tools.txt requirements/pip-tools.in
```

Comment:
Shouldn't need `--rebuild` on this line anymore since it's now on the previous one.